dev-python/python-efl/python-efl-1.20.0.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+MY_P=${P/_/-}
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_SUB_PROJECT="bindings/python"
+	EGIT_URI_APPEND="${PN}"
+	EGIT_REPO_URI="https://git.enlightenment.org/${EGIT_SUB_PROJECT}/${EGIT_URI_APPEND}.git"
+	inherit git-r3
+else
+	SRC_URI="https://download.enlightenment.org/rel/bindings/python/${MY_P}.tar.xz"
+fi
+
+inherit distutils-r1
+
+DESCRIPTION="Python bindings for Enlightenment Fundation Libraries"
+HOMEPAGE="https://www.enlightenment.org"
+
+LICENSE="|| ( GPL-3 LGPL-3 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples"
+
+RDEPEND=">=dev-libs/efl-${PV}
+	>dev-python/dbus-python-0.83[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	>=dev-python/cython-0.21[${PYTHON_USEDEP}]
+	doc? (
+		media-gfx/graphviz[python]
```

Comment:
If you dep on `[python]`, then you also need to use-dep on it. Please verify if that's really the case and if it is, then you'd either have to lower the ebuild to just py2.7 or use [any-of API of python-r1](https://wiki.gentoo.org/wiki/Project:Python/python-r1#Common_code_with_the_any-of_dependency_API) (which is harder).

Reply:
Indeed, [python] is not needed, dropping it